src/components/tools/text-tool.tsx

```diff
@@ -81,6 +93,24 @@ export default class TextToolComponent extends BaseComponent<IProps, IState> {
   private disposers: IReactionDisposer[];
   private prevText: any;
 
+  private plugins: Plugin[] = [
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+b", type: "bold" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+i", type: "italic" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+u", type: "underlined" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+.", type: "superscript" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "ctrl+,", type: "subscript" }),
+    this.makeOnKeyDownHandler({ slateType: ESlateType.block, key: "ctrl+alt+0", type: "deleted" }),
+    // this.makeOnKeyDownHandler({ slateType: ESlateType.mark, key: "alt+shift+t", type: "typewriter" }),
```

Comment:
Not sure this style will be enabled; but, the slate-mark name is no longer "typewriter", but "code".

By the way, "code" is a little odd in that it's the only slate `type` name that is used both in marks **AND** in blocks.